Decred Wallet/Storyboard.swift

```diff
@@ -21,8 +21,8 @@ enum Storyboard: String {
     case SeedBackup = "SeedBackup"
     case Settings = "Settings"
     case SecurityMenu = "SecurityMenu"
-    
     case CustomDialogs = "CustomDialogs"
+    case Receive = "Receive"
```

Comment:
Move to after `Overview`, before `Send`.

Reply:
https://github.com/raedahgroup/dcrios/pull/583/files#r376644842 is not resolved:
Move to after `Overview`, before `Send`.